test/integration2/dialects/mssql.spec.js

```diff
@@ -199,6 +199,38 @@ describe('MSSQL dialect', () => {
           });
         });
 
+        describe('unique table constraint with options object', () => {
+          const tableName = 'test_unique_constraint_options';
+          before(async () => {
+            await knex.schema.createTable(tableName, function () {
+              this.integer('x').notNull();
+              this.integer('y').notNull();
+            });
+          });
+
+          after(async () => {
+            await knex.schema.dropTable(tableName);
+          });
+
+          it('accepts indexName and constraint in options object', async () => {
+            const indexName = `UK_${tableName}_x_y`;
+            await knex.schema.alterTable(tableName, function () {
+              this.unique(['x', 'y'], {
+                indexName: indexName,
+                constraint: true,
+              });
+            });
+            expect(
```

Comment:
shouldn't this line be awaited?

Reply:
Copied the test above "unique table index with options object". Looks like other `except` tests are awaited. Want me to change?

Reply:
yes, please!

Reply:
Done 7a9de41